catchup/service.go

```diff
@@ -498,7 +498,7 @@ func (s *Service) periodicSync() {
 			sleepDuration = time.Duration(crypto.RandUint63()) % s.deadlineTimeout
 			continue
 		case <-time.After(sleepDuration):
-			if sleepDuration < s.deadlineTimeout {
```

Comment:
What is the purpose of periodicSync() if there is not network? 
Maybe return it before the loop?

Reply:
we still need it for the `unmatchedPendingCertificates`, which could come from the agreement protocol. I believe that it shouldn't happen, but it would be simpler just to let it pass through.